nomenclature/processor/data_validator.py

```diff
@@ -54,6 +74,19 @@ class DataValidator(Processor):
     criteria_items: List[DataValidationCriteriaBounds | DataValidationCriteriaValue]
     file: Path
 
+    @field_validator("criteria_items", mode="before")
+    def check_criteria(cls, v):
+        for criterion in v:
+            bounds = ["upper_bound", "lower_bound"]
+            values = ["value", "atol", "rtol"]
+            if any(bound in criterion for bound in bounds) and any(
+                value in criterion for value in values
+            ):
```

Comment:
A proposal to make this more readable...
```suggestion
            has_bounds = any(c in criterion for c in ["upper_bound", "lower_bound"])
            has_values = any(c in criterion for c in ["value", "atol", "rtol"])
            if has_bounds and has_values:
```

Reply:
Ah very nice, that one's much better 👍